internal/runners/packages/list.go

```diff
@@ -170,5 +178,16 @@ func newFilteredRequirementsTable(requirements []*gqlModel.Requirement, filter s
 		rows = append(rows, row)
 	}
 
-	return newTableOutput(rows, locale.T(fmt.Sprintf("%s_list_no_packages", nstype.String())))
+	// Sort the rows.
+	less := func(i, j int) bool {
+		a := rows[i].Pkg
+		b := rows[j].Pkg
+		if strings.ToLower(a) < strings.ToLower(b) {
+			return true
+		}
+		return a < b
+	}
+	sort.Slice(rows, less)
```

Comment:
Looks like you can drop `.sortByPkg()` since you moved the logic here.

Reply:
Yes, that was my intent. I don't see any `.sortByPkg()` calls anywhere.

Reply:
Ah, I see now you removed the entire file. I was looking for a `sortByPkg` method removal and not finding it.. :p